nengo_viz/components/raster.py

```diff
@@ -10,6 +10,9 @@
 class Raster(Component):
     def __init__(self, viz, config, uid, obj, n_neurons=None):
         super(Raster, self).__init__(viz, config, uid)
+        if getattr(obj, 'neuron_type') is not None:
```

Comment:
In that case, I think checking for None here is redundant. I will get rid of it.